examples/kwok/main_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kwok
+
+import (
+	"os"
+	"testing"
+
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/envfuncs"
+)
+
+var testenv env.Environment
+
+func TestMain(m *testing.M) {
+	testenv = env.New()
+	kwokClusterName := envconf.RandomName("kwok-cluster", 16)
+	namespace := envconf.RandomName("kwok-ns", 16)
+
+	testenv.Setup(
+		envfuncs.CreateKwokCluster(kwokClusterName),
```

Comment:
Do we support passing parameters to the underneath `kwokctl create cluster` command? For example, for a CI of custom scheduler, it may need to pass `--kube-scheduler-binary` specifically.

Reply:
@Huang-Wei we have `CreateWithConfig(kwokConfigFile string) ` for this use-case to pass any additional config. it follows the same pattern used for kind cluster creation.

Reply:
ah, that's sweet, then customization can be done via that single config file.